source/Redis/Fido.Redis.KVStore.pas

```diff
@@ -44,12 +46,18 @@   TRedisKVStore = class(TInterfacedObject, IKVStore)
     FRedisClient: IFidoRedisClient;
   private
     function FormatKey(const Key: string): string;
+
+    function Is1(const Value: Integer): Boolean;
```

Comment:
I think that a whole function to check that something is 1 is a bit overkill... If you want it, at least make it extendable for every number 😉

Reply:
No.
I like it as it is.
Even small 1 deserves respect.